docs/framework/wpf/advanced/drawing-formatted-text.md

```diff
@@ -56,35 +54,30 @@ Displayed text showing wordwrapping and ellipsis
 ### Converting Formatted Text to a Geometry  
  You can convert formatted text into <xref:System.Windows.Media.Geometry> objects, allowing you to create other types of visually interesting text. For example, you could create a <xref:System.Windows.Media.Geometry> object based on the outline of a text string.  
   
- ![Text outline using a linear gradient brush](./media/outlinedtext02.jpg "OutlinedText02")  
-Text outline using a linear gradient brush  
+ ![Text outline using a linear gradient brush](./media/typography-in-wpf/text-outline-linear-gradient.jpg)    
```

Comment:
both articles were using the same image?

Reply:
Yes. There were a few images used across a couple of articles.

Reply:
Ok.